templates/quality.html

```diff
@@ -0,0 +1,132 @@
+      </select>
+      <input type="text" class="form-control" name="stdout_value[]" value="{{ data.get('stdout_value', "") }}">
+    </div>
+  </div>
+  <div class="form-group row inspect-type-container">
+    <div class="col-md-3 col-form-label"><input class="inspect-include" type="hidden" name="stderr_include[]" value="{{'true' if data.get('stderr') else 'false'}}"><input class="inspect-type-checkbox mr-3" type="checkbox" {{'checked' if data.get('stderr')}}>Standard error</div>
+    <div class="input-group col-md-9 inspect-type-input {{'invisible' if not data.get('stderr')}}">
+      <select class="custom-select input-group-select" name="stderr_operator[]">
+        <option value="is" {{"selected" if data.get("stderr_operator", "") == "is"}}>is</option>
+        <option value="is-not" {{"selected" if data.get("stderr_operator", "") == "is-not"}}>is not</option>
+        <option value="contains" {{"selected" if data.get("stderr_operator", "") == "contains"}}>contains</option>
+        <option value="contains-not" {{"selected" if data.get("stderr_operator", "") == "contains-not"}}>does not contain</option>
+        <option value="empty" {{"selected" if data.get("stderr_operator", "") == "empty"}}>is empty</option>
+      </select>
+      <input type="text" class="form-control" name="stderr_value[]" value="{{ data.get('stderr_value', "") }}">
+    </div>
+  </div>
+  <div class="form-group">
+    <button type="button" class="copy-btn btn btn-outline-warning {{'d-none' if not show_add_btn}}">Add and continue on this page</button>
+    <button type="button" class="copy-btn btn btn-outline-primary {{'d-none' if not show_add_btn}}" data-submit="qa-form">Add and continue on next page</button>
+    <button type="button" data-toremove=".opt-form" class="rm-btn btn btn-outline-danger {{'d-none' if not show_rm_btn}}">Remove</button>
+  </div>
+{% endwith %}
+{%- endmacro %}
+{#- END: Quality Managment form macro -#}
+
+{#- BEGIN: Rendered content -#}
+{% block content %}
+  <script type="text/javascript">
+    $(function(){
+      $(document).on("change", ".inspect-type-checkbox", function(){
+        // Workaround: Checkboxes are only posted if checked, but we need a
+        // value per step to be able to aggregate step values by index.
+        // TODO: DRY with checkbox workaround in main.js
+        var checked = $(this).is(':checked');
+        $(this).prev("input.inspect-include").val(checked);
+
+        // Toggle inspection input fields visibility
+        // It would be nicer to disable the form inputs, but then they don't
+        // get posted which messes up the aggregation by index.
+        // Readonly would work, but only for input fields and not for select
+        $(this).parents(".inspect-type-container")
+            .find(".inspect-type-input")
+            .toggleClass("invisible");
+      });
+    });
+  </script>
+  {{ macros.progress_bar(value=40) }}
+  <hr>
+  <h1>How do you assure your software's quality?</h1>
```

Comment:
Is it possible for a project to use multiple tools, like `unittest`, `doctest`, and `Travis CI`.  It seems that you can only select one icon.

Reply:
Yep, that's the idea of the apparently rather ambiguous "Add and continue on this page" button. :)
This problem is already documented in https://github.com/in-toto/layout-web-tool/issues/25#issuecomment-329230813